class German:
    strings = {
        "no_cert_selected": "Hallo, {user_first_name}! Bitte wählen Sie ein Zertifikat aus dem /start-Menü aus und versuchen Sie es erneut. 🚀",
        "sign_error_retry": "Ein Fehler ist während der Signierung aufgetreten, erneuter Versuch... ({failed_sign_attempt}/{max_sign_attempt})",
        "signing_failed": "❌ Signierung für {file_name} fehlgeschlagen.",
        "unexpected_error": "Ein unerwarteter Fehler ist aufgetreten. Sollte dieses Problem weiterhin bestehen, wenden Sie sich bitte an den Support.",
        "ipa_ready": "✅ Ihr IPA wurde signiert und ist bereit.",
        "forgot_choice": "Es scheint, Sie haben vergessen, ein Zertifikat auszuwählen. Bitte gehen Sie zum /Startmenü, um eines zu wählen.",
        "file_lost": "Die Datei ging während des Signiervorgangs verloren.",
        "value_error": "Ein Wertfehler ist während des Signiervorgangs aufgetreten.",
        "permission_error": "Während des Signiervorgangs ist ein Berechtigungsfehler aufgetreten.",
        "signing_error": "Der Signiervorgang ist fehlgeschlagen.",
        "unknown_error": "Ein unerwarteter Fehler ist während des Signiervorgangs aufgetreten.",
        "select_certificate_button": "📝️ Zertifikat auswählen",
        "signing_in_progress": "📝 Signiere...",
        "modifying_in_progress": "🔧 Modifiziere...",
        "executing_command": "Ausführung: {command}",
        "modify_retry_error": "Ein Fehler ist beim Ändern aufgetreten, erneuter Versuch... ({failed_modify_attempt}/{max_modify_attempt})",
        "fnfe_err": "Die Datei konnte während des Signiervorgangs nicht gefunden werden.",
        "ve_err": "Ein Wertfehler ist während des Signiervorgangs aufgetreten.",
        "pe_err": "Ein Berechtigungsfehler ist während des Signiervorgangs aufgetreten.",
        "unknown_sign_err": "Ein unerwarteter Fehler ist während des Signiervorgangs aufgetreten."
    }
